rules/opentitan.bzl

```diff
@@ -120,17 +115,14 @@ sign_bin = rule(
             default = "//sw/host/rom_ext_image_tools/signer:rom_ext_signer",
             allow_single_file = True,
         ),
-        "_allowlist_function_transition": attr.label(
-            default = "@bazel_tools//tools/allowlists/function_transition_allowlist",
-        ),
     },
 )
 
 def _elf_to_disassembly_impl(ctx):
     cc_toolchain = find_cc_toolchain(ctx)
     outputs = []
     for src in ctx.files.srcs:
-        disassembly = ctx.actions.declare_file("{}.dis".format(src.basename))
+        disassembly = ctx.actions.declare_file("{}.elf.s".format(src.basename))
```

Comment:
super nit: (if this is really a thing then don't worry about it unless you're cherrypicking these commits) this change might be in the wrong commit.